website/pages/docs/configuration/storage/aerospike.mdx

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Need to update the JSON to reference this as well: https://github.com/hashicorp/vault/blob/f66c721e79fec9e51bd76841969db92a830cc6e7/website/data/docs-navigation.js#L67